Podcast/ExternalProfileViewController.swift

```diff
@@ -93,47 +104,96 @@ class ExternalProfileViewController: UIViewController, UITableViewDataSource, UI
         UIApplication.shared.statusBarStyle = .default
     }
     
-    func fetchUser(id: Int) {
+    func fetchUser(id: String) {
         
-        // Dummy data
-        let user = User()
-        user.firstName = "Paul"
-        user.lastName = "Dugg"
-        user.username = "doglover12"
-        user.numberOfFollowing = 100
-        user.numberOfFollowers = 4
+        let userRequest = UserByIDEndpointRequest(userID: id)
```

Comment:
change the name of this network request to something like FetchUserByIDEndpointRequest